test/1.4/statements/T_return.dml

```diff
@@ -0,0 +1,40 @@
+/*
+  © 2022 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.4;
+device test;
+import "testing.dml";
+
+typedef struct {
+    int i;
+    bool b;
```

Comment:
make one field `const` for more fun!

Reply:
Hey, that reveals a bug in how `codegen_exit` works! Not exclusive to this patch! I'll create a JIRA.

Reply:
Actually, wait, it _is_ exclusive to this patch! That wasn't obvious at first. Neat!

Reply:
I fixed this by banning methods from having (partially) const outputs, period. This needs to be tested against VP.